content/nginx/deployment-guides/amazon-web-services/high-availability-network-load-balancer.md

```diff
@@ -45,192 +43,214 @@ NGINX Plus provides both reverse‑proxy features and load‑balancing features
 
-2. Click the <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Add to registered </span> button above the table. The instances are added to the **Registered targets** table.
+2. Select the <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Add to registered </span> button above the table. The instances are added to the **Registered targets** table.
 
    <a href="/nginx/images/aws-nlb-load-balancer-targets-display.png"><img src="/nginx/images/aws-nlb-load-balancer-targets-display.png" alt="" width="1024" height="874" class="aligncenter size-full wp-image-54845" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>
 
-3. Click the <span style="background-color:#cccccc; font-weight:bolder; white-space: nowrap;"> Next: Review </span> button in the lower‑right corner of the window.
+3. Select the <span style="background-color:#cccccc; font-weight:bolder; white-space: nowrap;"> Next: Review </span> button. The **Step 4: Review** window opens.
 
 <span id="nlb-launch"></span>
-### Launching the AWS NLB
+### Launch the AWS NLB
 
-In the **Step 4: Review** window that opens:
+In this step you use the **Step 4: Review** window to verify settings and launch AWS NLB.
 
-1. Verify that the settings are correct. If so, click the <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Create </span> button in the lower‑right corner of the window. To change settings, click the <span style="background-color:#cccccc; font-weight:bolder; white-space: nowrap;"> Previous </span> button to go back to previous screens.
+1. In the **Step 4: Review** window verify that the settings are correct.
+     - If so, select the <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Create </span> button.
+     - If the settings are not correct, select the <span style="background-color:#cccccc; font-weight:bolder; white-space: nowrap;"> Previous </span> button to go back to a previous screen. Make required change(s) on previous screen(s). Then, return to the **Step 4: Review** window to select <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Create </span>.
 
    <a href="/nginx/images/aws-nlb-load-balancer-review.png"><img src="/nginx/images/aws-nlb-load-balancer-review.png" alt="" width="1023" height="789" class="aligncenter size-full wp-image-54843" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>
 
-2. The AWS NLB is provisioned. When the success message appears, click the <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Close </span> button to return to the **Load Balancers** dashboard.
+    The <span style="background-color:#3366cc; color:white; white-space: nowrap;"> Create </span> action provisions the AWS NLB. A success message displays when the provisioning operation finishes. It can take a few minutes to provision the AWS NLB.
```

Comment:
```suggestion
    The *Create* action provisions the AWS NLB. A success message displays when the provisioning operation finishes. It can take a few minutes to provision the AWS NLB.
```